plugins/woocommerce/src/Admin/Features/MultichannelMarketing/Init.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Multichannel Marketing Experience
+ *
+ * @package Woocommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Features\MultichannelMarketing;
+
+use Automattic\WooCommerce\Internal\Admin\Survey;
+use Automattic\WooCommerce\Admin\Features\Features;
+use Automattic\WooCommerce\Internal\Admin\WCAdminAssets;
+
+/**
+ * Contains logic for Multichannel Marketing.
+ */
+class Init {
+	/**
+	 * Option name used to toggle this feature.
+	 */
+	const TOGGLE_OPTION_NAME = 'woocommerce_multichannel_marketing_enabled';
+
+	/**
+	 * Determines if the feature has been toggled on or off.
+	 *
+	 * @var boolean
+	 */
+	protected static $is_updated = false;
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_filter( 'woocommerce_settings_features', array( $this, 'add_feature_toggle' ) );
+		add_action( 'update_option_' . self::TOGGLE_OPTION_NAME, array( $this, 'reload_page_on_toggle' ), 10, 2 );
+		add_action( 'woocommerce_settings_saved', array( $this, 'maybe_reload_page' ) );
+	}
+
+	/**
+	 * Add the feature toggle to the features settings.
+	 *
+	 * @param array $features Feature sections.
+	 * @return array
+	 */
+	public static function add_feature_toggle( $features ) {
+		$description = __(
+			'Enables the new WooCommerce Multichannel Marketing experience in the Marketing page',
+			'woocommerce'
+		);
+
+		$features[] = array(
+			'title' => __( 'Marketing', 'woocommerce' ),
```

Comment:
❓  / 💅  Shouldn't this title supposed to be "Multichannel Marketing"?

Reply:
I ran the copy with our design team (see slack thread p1658143055649809-slack-C03GUT0P0Q3) and it is fine. I think the idea is for the feature title "Marketing" to match with the menu name in the left navigation bar in wp-admin.

![image](https://user-images.githubusercontent.com/417342/179809535-5635b60f-cc1b-4e53-8c2b-82569fee0106.png)